### PR TITLE
Remove the exit for pylint errors

### PR DESCRIPTION
**Pull request by kianapaz**
Removed the specific selections of the pylint

### CODE REVIEW

**Comment by sfaigan** [comment]
Running a PEP8 compliant code formatter like Black should fix most if not all the linting errors

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -32,10 +32,8 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with flake8
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # The GitHub editor is 127 chars wide
+          flake8 . --count --max-line-length=127 --statistics
```
Comment (sfaigan) [inline .github/workflows/continuous_integration.yml:36]: 127 seems pretty long, where did you get this number?

**Review by sfaigan — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous_integration.yml (hunk 1 above)
Comment (kianapaz) [reply .github/workflows/continuous_integration.yml]: 127 is the GitHub editor length. I changed the length to now match the recommended length in Pycharm, 120 chars.

**Review by Ckoelewyn — APPROVED** [review]
Looks good, thanks for changing those other files so the linter passes!

**Review by sfaigan — APPROVED** [review]

**Pull request closed without merging**